docs/sources/auth/overview.md

```diff
@@ -18,6 +18,7 @@ Provider | Support | Role mapping | Team sync<br> *(Enterprise only)* | Active s
 [GitHub OAuth]({{< relref "github.md" >}})         | v2.0+ | - | v6.3+ | -
 [GitLab OAuth]({{< relref "gitlab.md" >}})         | v5.3+ | - | v6.4+ | -
 [Google OAuth]({{< relref "google.md" >}})         | v2.0+ | - | - | -
+[JWT]({{< relref "jwt.md" >}})                     | v7.4+ | - | - | -
```

Comment:
Need to be updated. This will probably not be included in 7.5 given we're close to stable release, but someone else might have another view of this. But might add `8.0+` for now rather than `7.4+`.